flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/io/AlternatingControllerTest.java

```diff
@@ -68,6 +68,22 @@
  */
 public class AlternatingControllerTest {
 
+	/**
+	 * Upon subsuming (or canceling) a checkpoint, channels should be notified regardless of whether UC controller is
+	 * currently being used or not. Otherwise, channels may capture in-flight buffers from an older checkpoint.
```

Comment:
> Otherwise, channels may capture in-flight buffers from an older checkpoint

Is this test actually checking for that? I do not see any buffer that would belong to an older checkpoint?

(I think I still do not understand this fix)

Reply:
Sorry, my bad, I didn't update the comment after updating the test. 
It checks whether the buffer (on the other channel) was captured or not. I'll update the comment.